proknow-sdk/Patient/Entities/EntityScorecardItem.cs

```diff
@@ -74,7 +74,7 @@ public override async Task DeleteAsync()
         //TODO--Add example to update a custom metric value for a scorecard
         public override async Task SaveAsync()
         {
-            var route = $"/workspaces/{_workspaceId}/entities/{_entityId}/metrics/sets/{Id}";
+            var route = $"/workspaces/{_workspaceId}/patients/{_entityId}/metrics/sets/{Id}";
```

Comment:
Entity scorecards are no longer a thing. They were replaced by Patient Scorecards. As as result, I think this class should to be moved up to the `proknow-sdk/Patient` level and renamed to something more appropriate, which is consistent with how we did things with the Python SDK.